<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QEditorWindow</class>
 <widget class="QMainWindow" name="QEditorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>monEditeur</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../.designer/backup/pixmap/centrejust.svg</normaloff>../../.designer/backup/pixmap/centrejust.svg</iconset>
  </property>
  <widget class="QMdiArea" name="centralwidget">
   <property name="viewMode">
    <enum>QMdiArea::SubWindowView</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>773</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Fichier">
    <property name="title">
     <string>&amp;Fichier</string>
    </property>
    <addaction name="action_Nouveau"/>
    <addaction name="action_Ouvrir"/>
    <addaction name="action_Enregistrer"/>
    <addaction name="action_Enregistrer_sous"/>
    <addaction name="separator"/>
    <addaction name="action_Quitter"/>
   </widget>
   <addaction name="menu_Fichier"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Nouveau"/>
   <addaction name="action_Ouvrir"/>
   <addaction name="action_Enregistrer"/>
   <addaction name="separator"/>
   <addaction name="action_Quitter"/>
  </widget>
  <widget class="QDockWidget" name="project">
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>200</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Projet</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QTreeWidget" name="projectTree">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>131</width>
       <height>192</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
  <widget class="QDockWidget" name="explorer">
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>200</height>
    </size>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>Explorateur de code</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <widget class="QTreeWidget" name="explorerTree">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>131</width>
       <height>151</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
  <action name="action_Nouveau">
   <property name="icon">
    <iconset>
     <normaloff>pixmap/document-new.png</normaloff>pixmap/document-new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Nouveau</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Ouvrir">
   <property name="icon">
    <iconset>
     <normaloff>pixmap/document-open.png</normaloff>pixmap/document-open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Ouvrir</string>
   </property>
  </action>
  <action name="action_Quitter">
   <property name="icon">
    <iconset>
     <normaloff>pixmap/exit.png</normaloff>pixmap/exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quitter</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="action_Enregistrer">
   <property name="icon">
    <iconset>
     <normaloff>pixmap/document-save.png</normaloff>pixmap/document-save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Enregistrer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Enregistrer_sous">
   <property name="text">
    <string>&amp;Enregistrer sous</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
